### ISSUE TITLE
[Proposal] Hand-pointer cursor for active elements of the application

### ISSUE THREAD

**Issue by Lisa24Jackson** [bkaankose/Wino-Mail#228]
In my opinion, for the active elements of the application, we should use not just a regular cursor but specifically a hand-pointer cursor. This would make app more intuitively clear that the element is active and it's considered better practice in terms of UI. A similar issue was present in the original version of the "Mail" app, which you're basing this on.

**Comment by bkaankose** [comment]
By "active element" you mean currently selected mail? If so, I don't remember such pointer change in the WMC. Do you mind sharing a few pictures for me to understand?

**Comment by Lisa24Jackson** [comment]
> By "active element" you mean currently selected mail? If so, I don't remember such pointer change in the WMC. Do you mind sharing a few pictures for me to understand?

Yes, that's correct. I meant the selected mail, menu items, etc. But I've decided to close this issue because I've realized that using a hand-pointer cursor like that isn't customary in e-mail clients. Alongside Microsoft Mail, I've also been using the web version of Gmail, and after the announcement about Mail being discontinued, I fully transitioned to Gmail's web version. Since it's a web version, buttons work like web links, hence the hand-pointer cursor is used for all interaction elements, and I've gotten used to this. It actually makes sense logically. However, upon reviewing interactions with Apple Mail, Thunderbird, etc., I see they use the standard pointer as well. Therefore, this question is removed. I'm sorry for bothering you.
